national-resources/estonia.md

```diff
@@ -0,0 +1,121 @@
+
+
+
+## National data sources
+<!--- A section to list and provide context to national data sources.  In the context of BY-COVID, a data source can be a repository that should include at least the metadata and, ideally, the data that might not be directly available when considering sensitive data. Also, repositories should have the capacity to share this data and, therefore, have a governance model in place on how to do it. It can also include registries of data sources important for the field, with a direct link to the original data sources to be able to request access to the data. --->
+From the onset of the pandemic in Estonia in March 2020, Estonia began openly sharing data related to SARS-CoV-2 testing. The first tests were conducted in February 2020, and as of January 2024, the test statistics are still regularly published on the dedicated [open data page](https://opendata.digilugu.ee/docs/#/) of the Health and Welfare Information Systems Centre of Estonia. The files must be downloaded for further use and should not be directly integrated into websites. The files contain data on:
+* [Covid-19 vaccination files](https://opendata.digilugu.ee/docs/#/en/opendata/covid19/vaccination/readme) (updated once per day):
+  - Nationwide data
+  - Coverage by age groups
+  -  Coverage by county
+  -  Coverage by age group and county
+  -  Coverage by age group by municipality
+  -  Coverage by age group by administrative unit
+  -  Statistics about institutions which carry out vaccinations
+* [SARS-CoV-2 testing files](https://opendata.digilugu.ee/docs/#/en/opendata/covid19/test/readme) (updated once per day):
+  - Nationwide positive cases
+  - Test results
+  - Average age of cases
+  - Location-based statistics
+  - County total statistics
+* [Covid-19 hospitalisations files](https://opendata.digilugu.ee/docs/#/en/opendata/covid19/hospitalization/readme) (last data shared on 02.05.2023).
+  - Average age of patients
+  - Time of patient hospitalisation
+  - Age and gender profile of patients
+  - Timeline of hospitalisation statistics.
+
+All the data above is shared both in JSON and CSV formats.
+
+## Regulations
+<!--- Ethical and legal regulations in the country, committees, etc --->
+In Estonia, a research plan that involves human subjects needs to be approved by an ethics board. The two general ethics boards that researchers can apply to are [(Research Ethics Committee of the University of Tartu](https://ut.ee/en/node/113848) and [Research Ethics Committee of the National Institute for Health Development](https://en.tai.ee/en/about-us/tallinn-medical-research-ethics-commitee).
```

Comment:
An extra "(" in the first part of the link. Should be: [Research Ethics Committee of the University of Tartu](https://ut.ee/en/node/113848)